.gitignore

```diff
@@ -25,4 +25,6 @@ ehsm_kms_service/package-lock.json
 ehsm_kms_service/logs/*/
 ehsm_kms_service/*.so
 
+.history/
+.vscode/
```

Comment:
change the commit title as
"
add support for API key and AppId enrollment

The new application (ehsm-kms_enroll_app) is responsible for obtaining
a valid access secret (API key and AppId) from ehsm-core enclave.

And the access secret is displayed only during the enrollment process, you
cannot query the access secret in subsequent crypto operations. Therefore,
you must back up your access secret, if they're leaked or lost, you must create
another one. 

Currently, the access secrets are transferred w/o secure channel protection, and
later we will establish a secure channel via the sgx remote attestation to assure its
security.
"